TableViewMarch2016/Base/TwoLabelCell.swift

```diff
@@ -40,8 +40,7 @@ class TwoLabelCell: UITableViewCell, Configurable {
     fatalError("init(coder:) has not been implemented")
   }
   
-  func config(withItem item: Any) {
```

Comment:
I'm wondering if it's better  for the `item` type to be `LabelsPresentable` instead of `Any`. Then it will be no need for the `guard`